ga4gh/prototools.py

```diff
@@ -0,0 +1,400 @@
+"""
```

Comment:
I'm not convinced that we need this module. It's only really used in the ``test_schemas`` test suite, and this was only really intended to test our custom JSON serialisation/deserialisation. We're now using protobuf's native support for this, and I think we should just trust that it works as advertised. The Validator is used in the exceptions module, but this doesn't really do anything right now.

So, I would vote for removing this module entirely, and also remove test_schemas.py.